tests/vmcheck/test-basic.sh

```diff
@@ -59,3 +59,17 @@ echo "ok status doesn't require root"
 # Also check that we can do status as non-root non-active
 vm_cmd runuser -u bin rpm-ostree status
 echo "ok status doesn't require active PAM session"
+
+# Add metadata string containing EnfOfLife attribtue
+META_ENDOFLIFE_MESSAGE="this_is_a_test"
```

Comment:
ah, the main reason is probably I dont know how to handle with quotes related syntax in shell , thus trying to stay away from it ( a bad habit from school I would say :( ). Currently working on to add it.